recipe/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - python
     - cf_units
     - fiona
+    - gdal
```

Comment:
Sorry. the `osgeo` module is part of the python bindings, correct? We are building now `libgal` and `gdal` (the python bindings) separately. So you need this PR. I'd just pin to `gdal 2.1.*` to avoid conflicts. (`fiona` does not work with gdal 2.2.* yet.)